src/Entity/Template.php

```diff
@@ -708,12 +708,7 @@ public function setContentType(ContentType $contentType = null)
         return $this;
     }
 
-    /**
-     * Get contentType.
-     *
-     * @return \EMS\CoreBundle\Entity\ContentType
-     */
-    public function getContentType()
+    public function getContentType(): ContentType
```

Comment:
Did you try to create a brand new template? I'm afraid that the field migth be null in that case

Reply:
Fixed add a giveContentType and reverted this change